README.rst

```diff
@@ -38,22 +37,21 @@ Installation
 
 Install via ``pip``:
 
-.. code-block:: bash
+::
 
     $ pip install dropbox
 
-
 Install from source:
 
-.. code-block:: bash
+::
 
     $ git clone git://github.com/dropbox/dropbox-sdk-python.git
     $ cd dropbox-sdk-python
     $ python setup.py install
 
 After installation, to get started, open a Python console:
 
-.. code-block:: python
+::
```

Comment:
No, but the `code-block` syntax is a Sphinx extension that's not available on PyPi. PyPi will botch the formatting on the package page (and will likely screw up parsing the classifiers) if these are included in the `long_description`.